sycl/include/sycl/handler.hpp

```diff
@@ -2289,8 +2289,7 @@ class __SYCL_EXPORT handler {
                   "Invalid accessor target for the copy method.");
     static_assert(isValidModeForDestinationAccessor(AccessMode),
                   "Invalid accessor mode for the copy method.");
-    static_assert(is_device_copyable<T_Src>::value,
-                  "Pattern must be device copyable");
+    // TODO: Require is_device_copyable when vec is device-copyable.
```

Comment:
```suggestion
    // TODO: Require static_assert with is_device_copyable when vec is device-copyable.
```
Maybe something like that?

Reply:
I have changed it to mention `static_assert`, but then I think "require" becomes redundant. Do you think the new naming is better?